schemas/iso19139/src/main/plugin/iso19139/index-fields/index.xsl

```diff
@@ -114,9 +114,10 @@
                   select="gmd:language[1]/gmd:LanguageCode/
```

Comment:
With this change it does not look like mainLanguageCode is used anymore so maybe it can be removed?